framework/entity/ServerEntities.xml

```diff
@@ -196,6 +196,7 @@ along with this software (see the LICENSE.md file). If not, see
         <field name="username" type="text-short"/>
         <field name="password" type="text-medium" encrypt="true"/>
         <field name="emailAddress" type="text-medium"/>
+        <field name ="authTokenCmd" type="text-medium"/>
```

Comment:
Please add a description to `authTokenCmd` and remove the extra space there

Reply:
Added description to authTokenCmd.
Removed the extra space.
Thanks for pointing it out!

Reply:
You're welcome!